pkg/reconciler/delivery/table_test.go

```diff
@@ -91,3 +132,35 @@ func Configuration(namespace, name string, co ...ConfigOption) *v1.Configuration
 	c.SetDefaults(context.Background())
 	return c
 }
+
+// this type is simply a convenient alias, see customRoute funtion below for its purpose
+type pair struct {
+	name  string
+	value int64
+}
+
+// customRoute extracts some verbiage from the table tests to make them more concise
+func customRoute(field string, nameValuePairs ...pair) RouteOption {
```

Comment:
nit: suggest a name that is more explicit, such as withTraffic

Reply:
fixed. I will wait for Victor's opinion before merging